### PR TITLE
Improve WIA Cropping

### PR DESCRIPTION
**Pull request by adghayes**
## Overview

#4025 

## Demo Video or Screenshot

| Before  | After |
| ------------- | ------------- |
| Screen Shot 2023-11-13 at 4 42 03 PM  | Screen Shot 2023-11-16 at 2 03 45 PM  |
| Screen Shot 2023-11-13 at 4 40 33 PM |  Screen Shot 2023-11-16 at 1 37 11 PM  | 

## Testing Plan

Manual mostly, updated snapshots / fixtures.

## Checklist

- [ ] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.

- [x] I have added a screenshot and/or video to this PR to demo the change
- [x] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

File: libs/hmpb/layout/src/layout.ts [hunk 1]
```diff
@@ -1687,7 +1687,7 @@ function layOutBallotHelper({
         bottom: 1,
         left: 1,
         right: 9,
-        top: 2,
+        top: 1,
```
Comment (adghayes) [inline libs/hmpb/layout/src/layout.ts:1690]: We were cropping a little high on the VxBallots, see difference between highlight and desired area:

Screen Shot 2023-11-13 at 4 42 13 PM

The cropping too low is dealt with in the interpreter. The above is correct - it says `1` and we want `1`. That off-by-one issue is in the interpreter.

File: libs/types/src/cdf/ballot-definition/fixtures.ts
Comment (adghayes) [inline libs/types/src/cdf/ballot-definition/fixtures.ts:1]: This is not necessary but thought I'd make it consistent.

File: libs/ballot-interpreter/src/hmpb-rust/layout.rs [hunk 2]
```diff
@@ -33,11 +33,9 @@ fn build_option_layout(
     let column_offset = -(grid_layout.option_bounds_from_target_mark.left as i32);
     let row_offset = -(grid_layout.option_bounds_from_target_mark.top as i32);
     let width: GridUnit = grid_layout.option_bounds_from_target_mark.left
-        + grid_layout.option_bounds_from_target_mark.right
-        + 1;
+        + grid_layout.option_bounds_from_target_mark.right;
     let height: GridUnit = grid_layout.option_bounds_from_target_mark.top
-        + grid_layout.option_bounds_from_target_mark.bottom
-        + 1;
+        + grid_layout.option_bounds_from_target_mark.bottom;
```
Comment (adghayes) [inline libs/ballot-interpreter/src/hmpb-rust/layout.rs:38]: Here's the off-by-one issue. We're adding in 1 to the width and the height, in addition to the `top + bottom` or `left + right` specified by the `gridLayouts`. I imagine the original thought here was to account for the size of the bubble itself? But we don't need to do that, the measurements are from the center of the bubble in all directions.

Comment (jonahkagan) [reply libs/ballot-interpreter/src/hmpb-rust/layout.rs:38]: Weird, I don't know why we would have done that! Nice catch

**Review by jonahkagan — APPROVED** [review]

**Pull request merged**